src/machine/machine_stm32.go

```diff
@@ -5,14 +5,30 @@ package machine
 // Peripheral abstraction layer for the stm32.
 
 import (
+	"device/arm"
 	"device/stm32"
 )
 
 type GPIOMode uint8
 
+const SystemClockSpeed = 72000000
```

Comment:
This is actually the system clock frequency, not the CPU frequency. That value could in theory change based on the user's clock register settings. 72Mhz is such as typical number used that it needed go be made into some kind of const, right? Maybe we need to have a more global group of consts with different common speed values.

Reply:
What's exactly the difference between system clock and CPU frequency? Aren't they both just the CPU frequency?

Reply:
The system clock at ARM can be based on several different sources, including the CPU freq. and can be changed at runtime.

The CPU frequency for AVR is set by fuses that can only be changed when flashing the chip and does not have another source for the system clock as far as I know.

If any change is called for to unify terms it might be to have AVR use the term `SystemClockSpeed` but probably not worth it, since typical AVR docs refer to this as CPU frequency.

Reply:
I've been looking into `SystemClockSpeed` but it appears to be specific to CMSIS, and it doesn't seem to be used much. Looks like it is intended to easily communicate the current clock frequency across the system.
Cortex-M devices appear to be more flexible in their clock configurations, and can sometimes change the CPU frequency at runtime (like in the case of stm32, but not in the case of nrf). I think changing the frequency makes sense in some specific circumstances (like variable power sources), but in general I can't see a reason why you wouldn't want to run at the max clock speed at all times. By using a high CPU frequency, computations are faster to complete and the chip can go to sleep faster which saves energy.
Please correct me if I'm wrong, but in this light, the `SystemClockSpeed` global variable as used in CMSIS doesn't seem to make a lot of sense in the context of TinyGo. The only thing that is relevant is the (`const`) CPU frequency used on any particular configuration, which may depend on the board (whether there is a crystal available, for example). And thus it would be equivalent to `CPU_FREQUENCY` on the AVR.

Reply:
OK, changed.